deepchem/models/callbacks.py

```diff
@@ -4,6 +4,13 @@
 
 import tensorflow as tf
 import sys
+from deepchem.metrics import Metric
+
+
+class StopIteration(Exception):
```

Comment:
StopIteration is a [built in exception class](https://docs.python.org/3/library/exceptions.html#StopIteration).  We don't need to define our own class that shadows it.

Reply:
Oh good, I was not aware of this.

Reply:
Done.